technologies/css.md

```diff
@@ -727,6 +728,7 @@ But in the following example we would not use it:
 [Nature playbook(private)]: https://github.com/nature/playbook
 [bem]: https://github.com/springernature/frontend/practices/bem-css.md
 [a hybrid of OOCSS and Atomic CSS]: https://github.com/springernature/frontend/practices/mosaic-css.md
```

Comment:
Sorry, I meant change it to something mentioning "utility classes" not Atomic, as we agreed that this new fangled thing called Atomic CSS at acss.io was a possible sauce of confusion ;)

Also I think the link should be;
https://github.com/springernature/frontend-playbook/blob/master/practices/mosaic-css.md